src/commands/deploy/verifyAppSettings.ts

```diff
@@ -32,6 +44,8 @@ export async function verifyAppSettings(context: IActionContext, node: SlotTreeI
         if (node.site.isLinux) {
             const remoteBuildSettingsChanged = verifyLinuxRemoteBuildSettings(context, appSettings.properties, bools);
             updateAppSettings ||= remoteBuildSettingsChanged;
+        } else if (isNodeV4Plus(options)) {
```

Comment:
This should be its own `if` case instead of in an `else if`. The new Node.js programming model can be run on both windows and linux

Reply:
Ah doh, obviously.